docs/libcurl/opts/CURLOPT_ALTSVC_CTRL.3

```diff
@@ -69,7 +69,9 @@ Accept alternative services offered over HTTP/3. This will only be used if
 libcurl was also built to actually support HTTP/3, otherwise this bit will be
 ignored.
 .SH DEFAULT
-0. No Alt-Svc treatment.
+Depends on what protocol support that is built-in. H1 is always there. H2 if
+libcurl is built with HTTP/2 support and H3 as well if HTTP/3 support is
+present.
```

Comment:
That is explained above how about just `CURLALTSVC_H1  | CURLALTSVC_H2 | CURLALTSVC_H3`

Reply:
Ah I see what's happening. CURLOPT_ALTSVC is causes altsvc initialized with default `CURLALTSVC_H1 | CURLALTSVC_H2 | CURLALTSVC_H3`. However altsvc would not be enabled without either CURLOPT_ALTSVC or 
CURLOPT_ALTSVC_CTRL therefore I think 0 is the default. It probably could be explained better though,
"0. No Alt-Svc treatment. If CURLOPT_ALTSVC was set then alt-svc ctrl defaults to CURLALTSVC_H1 | CURLALTSVC_H2 | CURLALTSVC_H3".

Reply:
I took a stab at an update. What do you say now?